src/styles/colors.scss

```
@@ -0,0 +1,4 @@
+$black: #000;
+$white: #fff;
```

```suggestion
$black: #000000;
$white: #ffffff;
```
WDYT? just to be consistent with the format of all the colors 🎨